task4-6/src/main/resources/application.properties

```diff
@@ -0,0 +1,13 @@
+spring.datasource.url=
+spring.datasource.username=
+spring.datasource.password=
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.hikari.maximum-pool-size=20
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+server.error.include-message=always
+server.error.include-binding-errors=always
+
+bucket.capacity=600
+bucket.interval-minute=1
```

Comment:
End of file https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log